src/java/org/apache/cassandra/net/MessagingService.java

```diff
@@ -264,6 +239,36 @@ public static List<Version> supportedVersions()
             return Collections.unmodifiableList(versions);
         }
     }
+    // Maintance Note:
+    // Try to keep Version enum in-sync for testing.  By having the versions in the enum tests can get access without forcing this class
+    // to load, which adds a lot of costs to each test
+    @Deprecated
+    public static final int VERSION_30 = 10;
```

Comment:
RE the comment above, can we not just make this...
```suggestion
    public static final int VERSION_30 = Version.VERSION_30.value;
```
...and the same for the others?

Reply:
our usage is with `switch`, which gets mad if you do that... I tried and had to revert =D